integration-tests/ts/src/main.ava.ts

```diff
@@ -0,0 +1,261 @@
+/**
+ * This tests the behavior of the standard FT contract at
+ * https://github.com/near/near-sdk-rs/tree/master/examples/fungible-token
+ *
+ * Some advanced features of near-workspaces this shows off:
+ *
+ * - Cross-Contract Calls: the "defi" contract implements basic features that
+ *   might be used by a marketplace contract. You can see its source code at the
+ *   near-sdk-rs link above. Several FT methods make cross-contract calls, and
+ *   these are tested below using this "defi" contract.
+ *
+ * - Complex transactions: to exercise certain edge cases of the FT standard,
+ *   tests below initiate chains of transactions using near-workspaces's transaction
+ *   builder. Search for `batch` below.
+ */
+import { Worker, NearAccount, captureError, NEAR, BN } from 'near-workspaces';
+import anyTest, { TestFn } from 'ava';
+
+const STORAGE_BYTE_COST = '1.5 mN';
+
+async function init_ft(
```

Comment:
Why are you using these helper functions? You can just move this logic to `beforeEach`.

Reply:
Have adjusted in most recent commit. Note I sourced this code from: https://github.com/near/workspaces-js/blob/d7c4f78b6b564082ee708722dd519ab7e9cb81da/__tests__/04.cross-contract-calls-with-fungible-token.ava.ts

So that repo would also have to be updated.